service/src/main/resources/application.yaml

```diff
@@ -132,7 +132,8 @@ scheduler:
       cronGroup: "NIGHTLY_CRON"
       description: 'Finds upcoming hearings'
       jobClass: 'uk.gov.hmcts.reform.fpl.jobs.UpcomingHearingsFinder'
-      cron: ${UPCOMING_HEARINGS_CRON:0 0 2 ? * MON-FRI *}
+      cron: ${UPCOMING_HEARINGS_CRON:0 0/15 * ? * MON-FRI *}
```

Comment:
it is not needed now, upgrade was done on Friday, so there should be already job run. If we still need to change cron expression it should be rather done via UPCOMING_HEARINGS_CRON in flux

Reply:
ok. Thanks Tomek ... I will close this PR